apps/designer/app/designer/features/style-panel/shared/style-info.ts

```diff
@@ -44,16 +44,23 @@ export type StyleInfo = {
   [property in StyleProperty]?: StyleValueInfo;
 };
 
-export type StyleSource = "local" | "preset";
+export type StyleSource = "local" | "remote" | "preset";
```

Comment:
maybe a better name should have been StyleOrigin here, so that `styleSource = {origin: 'local'}`